Dockerfile.microshift

```
@@ -8,10 +8,10 @@ ENV VERSION=rhel9 COMMIT=unset
 RUN ./hack/build-go.sh
 WORKDIR /
 
-FROM registry.ci.openshift.org/ocp/4.22:base-rhel9
+FROM registry.ci.openshift.org/ocp/5.0:base-rhel9
 RUN dnf install -y util-linux && dnf clean all && \
     mkdir -p /usr/src/multus-cni/bin
-COPY --from=rhel9 \
+COPY --from=builder \
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)